src/core/connection.c

```diff
@@ -1737,7 +1739,8 @@ QuicConnStart(
     _In_ QUIC_CONFIGURATION* Configuration,
     _In_ QUIC_ADDRESS_FAMILY Family,
     _In_opt_z_ const char* ServerName,
-    _In_ uint16_t ServerPort // Host byte order
+    _In_ uint16_t ServerPort, // Host byte order
+    _In_ uint32_t StartFlags
```

Comment:
We should use a well-defined enum for flags, including a definition for NONE instead of passing zero other places.